src/s2/base/port.h

```diff
@@ -290,8 +290,8 @@ inline void sized_delete_array(void *ptr, size_t size) {
 
 #endif
 
-// defines __BYTE_ORDER for MSVC
-#ifdef _MSC_VER
+// defines __BYTE_ORDER for MSVC or MINGW
+#if defined(_MSC_VER) || defined(__MINGW32__)
```

Comment:
`_MSC_VER` was probably a mistake. If `_WIN32` works for both windows and MinGW, let's use that.